tensorboard/plugins/mesh/mesh_demo.py

```diff
@@ -0,0 +1,115 @@
+"""Simple demo which displays constant 3D mesh."""
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Demo application for Mesh Plugin."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import numpy as np
+import tensorflow as tf
+import trimesh
+
+from tensorboard.plugins.mesh import summary as mesh_summary
+
+tf.flags.DEFINE_string('logdir', '/tmp/mesh_demo',
+                       'Directory to write event logs to.')
+tf.flags.DEFINE_string('sample_meshes', '',
+                       'Path to folder with PLY files or string of comma '
+                       'separated PLY files to visualize.')
+tf.flags.DEFINE_integer('batch_size', 1,
+                        'Size of the batch. All provided PLY files will be '
+                        'divided into batch_size groups and used at different '
+                        'steps during training.')
+FLAGS = tf.flags.FLAGS
+
+tf.compat.v1.disable_v2_behavior()
```

Comment:
I’m inferring from this line that you’ve tested this in both TF 1.x and
TF 2.0, and it works in both when V2 behavior is disabled. If so, great!
If it doesn’t work in TF 2.0 even with this line, then please fail
upfront.

Reply:
I haven't tested it in TF2.0 yet. How do I force it in g3?